assets/js/blocks/mini-cart/edit.tsx

```diff
@@ -80,7 +82,7 @@ const Edit = ( { attributes, setAttributes }: Props ): ReactElement => {
 		className: 'wc-block-mini-cart',
 	} );
 
-	const isSiteEditor = isSiteEditorPage( select( 'core/edit-site' ) );
+	const isSiteEditor = postType === undefined || postId === undefined;
```

Comment:
Why are you just using the presence of a `postType` or `postId` value as an indicator of whether this is in the site editor environment?

Reply:
There are a few places on GB where they use the same, such as [within the post-comments-form](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/post-comments-form/form.js#L63), so used that as an inspiration, but I agree this doesn't seem to be a rock-solid foundation to build a house on. Is there a recommended approach for checking if a site is in editor environment?

Reply:
Yup, as @nefeline says, we used the Post Comments Form block as a reference here. And even though the variable is named `isSiteEditor`, I think what's relevant here is whether a template is being edited or not.

I guess we could rewrite this code to use the same method that we are introducing for Product Query:

```TS
const isTemplate = useSelect( ( select ) => {
	const store = select( 'core/edit-site' );
	const editedPostType = store?.getEditedPostType();

	return editedPostType === 'wp_template';
}, [] );
```

Does that sound like a good approach?

Reply:
I went ahead and made the change in e9defb0358f9b0807cbc32225efaa949adebdf9e.

Reply:
> Does that sound like a good approach?

👍